src/main/java/mil/dds/anet/ws/GraphQLWebService.java

```diff
@@ -0,0 +1,81 @@
+package mil.dds.anet.ws;
+
+import io.leangen.graphql.spqr.spring.web.dto.GraphQLRequest;
+import java.security.Principal;
+import java.util.Map;
+import javax.security.auth.Subject;
+import mil.dds.anet.beans.AccessToken;
+import mil.dds.anet.database.AccessTokenDao;
+import mil.dds.anet.graphql.outputtransformers.ResourceTransformers;
+import mil.dds.anet.resources.GraphQLResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/graphqlWebService")
+public class GraphQLWebService {
+  public class GraphQLWebServicePrincipal implements Principal {
```

Comment:
Why not extract this as a generic AccessTokenPrincipal, that can also be used by the NVG web service?